index.html

```diff
@@ -0,0 +1,79 @@
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/6887534a1e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Library</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-11 mx-auto text-center my-5">
+                <form class="my-5" id="customer-form">
+                    <h3 class="text-capitalize">My Library</h3>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-book"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize title" id="title" placeholder="title">
+                    </div>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-user"></i></span>
+                        </div>
+                        <input type="text" class="form-control text-capitalize author" placeholder="author" id="author">
+                    </div>
+                    <div class="input-group my-3">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text"><i class="fas fa-file-alt"></i></span>
+                        </div>
+                        <input type="number" class="form-control text-capitalize pages" placeholder="number of pages"
+                            id="pages">
+                    </div>
+                    <div class="read d-flex">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text radius-none"><i class="fas fa-book-reader"></i></span>
+                        </div>
+                        <div class="custom-control custom-radio custom-control-inline pt-1 ml-4">
+                            <input type="radio" id="customRadioInline1" name="customRadioInline1"
+                                class="custom-control-input" value="yes">
+                            <label class="custom-control-label" for="customRadioInline1">Yes</label>
+                        </div>
+                        <div class="custom-control custom-radio custom-control-inline pt-1">
+                            <input type="radio" id="customRadioInline2" name="customRadioInline1"
+                                class="custom-control-input" value="no">
+                            <label class="custom-control-label" for="customRadioInline2">No</label>
+                        </div>
+                    </div>
```

Comment:
[OPTIONAL]
Make one of these a default value by marking it checked. I suggest the `customeRadioInline2` as default checked value.